sbom-utility-scripts/scripts/add-image-reference-script/README.md

```diff
@@ -0,0 +1,146 @@
+# Add image reference script
+
+The script aims to enrich the SBOM file with additional information about the output image used in the build process.
+Based on a input SBOM type, the script updates certain fields with the image reference information. This is needed
+to provide a complete SBOM file that can be used for further analysis.
+
+## Usage
+
+```bash
+python add_image_reference.py \
+    --input-file ./input-sbom.json \
+    --output-file ./updated-sbom.json \
+    --image-url quay.io/foo/bar/ubi8:1.1 \
+    --image-digest sha256:011ff0cd8f34588d3eca86da97619f7baf99c8cc12e24cc3a7f337873c8d36cc
+```
+The script stores the updated SBOM in the output path provided.
+
+## List of updates
+### SPDX
+The script updates the following fields in the SPDX SBOM:SHA256
+- `packages` - the script adds a new package with the image reference information
+- `relationships` - the script adds a new relationship between the package and the image reference
+- `name` - the script adds the image reference as a name
+
+The logic of adding new packages and updating replationships is visualized in the following diagram:
+```
+cachi2 SPDX SBOM:
```

Comment:
nitpick: it's not clear if cachi2 SBOMs will look like this, let's just call it an "example SPDX SBOM"

Reply:
Renamed